extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -570,25 +570,151 @@ describe('Project: sqlproj content operations', function (): void {
 
 	});
 
-	it('Should not allow adding duplicate file/folder entries in sqlproj', async function (): Promise<void> {
+	it('Should not allow adding duplicate file/folder entries in new sqlproj by default', async function (): Promise<void> {
 		projFilePath = await testUtils.createTestSqlProjFile(baselines.newProjectFileBaseline);
 		const project: Project = await Project.openProject(projFilePath);
 		const fileList = await testUtils.createListOfFiles(path.dirname(projFilePath));
 
-		// verify first entry in list is a folder
-		const existingFolderUri = fileList[0];
+		// 1. Add a folder to the project
+		const existingFolderUri = fileList[2];
```

Comment:
I'm not fond of magic numbers, but I didn't want to butcher the code too much for the sake of diff. The array is as following:
```javascript
[
  '', // [0]
  'script1.sql', // [1]
  'folder1', // [2]
  'folder1\script1.sql', // [3]
  ...
]
```

Reply:
You don't want to hardcode either. What we want here is to actually find first/second/third/etc. folder/file item in the array using `fs.stat` or something. That will be a slightly bigger code change, which I didn't want to include in this changeset, as it already takes 3+ days to get this in and we would like to move a little bit faster than that. I will keep it this way for now just for the sake of making progress and you can add a workitem to improve this within your team.